AndelaOSP/andela-societies-frontend
# Ft 157622944 fetch society in review activities

## Resolves [157622944](https://www.pivotaltracker.com/story/show/157622944)
## Description
- Fetch secretary's profile information to identify their society
- Load only logged activities that are **in review** in the verify activities page for the society secretary for the society to which they belong
## How to test
- Once logged in, navigate to the verify activities page, you should only see logged activities that are in review for your society

### Note
- The verify activities page will be accessible to only society secretaries and success department members once roles have been implemented
